parallel-lint

```diff
@@ -38,6 +38,7 @@ if (!defined('PHP_VERSION_ID') || PHP_VERSION_ID < 50400) {
 $autoloadLocations = [
     getcwd() . '/vendor/autoload.php',
     getcwd() . '/../../autoload.php',
+    __DIR__ . '/vendor/autoload.php',
```

Comment:
I have a similar case.
If you call the script from outside your project folder, the same error message appears.
e.g. `my-project-in-subfolder/vendor/bin/parallel-lint`

This would fix it:
```suggestion
    __DIR__ . '/vendor/autoload.php',
    __DIR__ . '/../../autoload.php',
```

Reply:
Yes, there are basically more issues with the autoloading. PR #2 was also a step in the right direction, but a more structural, stable solution is needed.